README.md

```diff
@@ -111,6 +111,13 @@ with lock:
 # And released here.
 ```
 
+All log messages by this library are made using the *DEBUG* level. On how to
```

Comment:
All log messages by this library are made using the *DEBUG* level, under the `filelock` name. 

👍 Forgot to document the name.

Reply:
Done 🚀